src/inspirehep-holdingpen-js/holdingpen/holdingpen.services.js

```diff
@@ -32,12 +32,19 @@
           return {
 
             getRecord: function (vm, workflowId) {
+              var service = this;
               $http.get('/api/holdingpen/' + workflowId).then(function (response) {
                 vm.record = response.data;
                 if (vm.record._workflow.data_type === 'authors') {
                   $('#breadcrumb').html(vm.record.metadata.name.value);
                 } else {
-                  $('#breadcrumb').html(vm.record.metadata.titles[0].title.substring(0, 70) + '...');
+                  var title;
+                  if (service.hasCrawlErrors(vm.record)) {
+                    title = vm.record._extra_data.crawl_errors.file_name;
+                  } else {
+                    title = vm.record.metadata.titles[0].title;
+                  }
+                  $('#breadcrumb').html(title > 70 ? title.substring(0, 70) + '...' : title);
```

Comment:
can we just do it with css? :) ``substring`` can be tricky for weird chars

Reply:
well that's what was there. but if you think we should, I can update